doc/basics.md

```diff
@@ -98,10 +102,13 @@ for (dom::object car : cars) {
 }
 ```
 
+
+
 JSON Pointer
 ------------
 
-The simdjson library also supports JSON pointer, letting you reach further down into the document:
+The simdjson library also supports JSON pointer through the at() method, letting you reach further
```

Comment:
Maybe we could make JSON pointer a link to https://tools.ietf.org/html/rfc6901 or some other ressource, when we first mention the term. I don't think we are assuming that everyone knows the spec. or even that there is a spec.

Reply:
Good idea, added.